views/group/show.pug

```diff
@@ -3,13 +3,28 @@ extends ../layouts/main
 block content
   .group-show
     .columns
-      .column
-        h1.title= group.name
       .column.is-narrow
-        if isOwner
-          button.button.is-danger(type="button", onclick=`deleteGroup(${group.id})`) 
-            i.fas.fa-trash-alt
-            span &nbsp;Törlés
+        h1.title= group.name
+      .column.has-text-right
+        .dropdown.is-right
+          .dropdown-trigger
+            button.button(aria-haspopup="true", aria-controls="dropdown-menu")
+              span.icon
+                i.fas.fa-sliders-h
+              span.icon.is-small
+                i.fas.fa-angle-down
+          .dropdown-menu(id="dropdown-menu", role="menu")
+            .dropdown-content
+              a.dropdown-item(onclick=`copyGroup(${group.id})`)
```

Comment:
Since this is an anchor tag, can we just set it's `href` attribute to `/groups/${group.id}/copy`, rather than navigating from JS code?